Assets/Scripts/UI/MouseOverFurnitureTypeText.cs

```diff
@@ -31,19 +31,26 @@ void Start()
             return;
         }
     }
-	
+
     // Update is called once per frame
     void Update()
     {
         Tile t = mouseController.GetMouseOverTile();
 
-        string s = "NULL";
```

Comment:
Becouse Of This Edit, It Will Not Display Anything Instead Of The Intended "NULL"